src/Image.ts

```diff
@@ -311,6 +319,41 @@ export class Image {
       this.data = data;
     }
   }
+  /**
+   * Returns normalized resolution in pixels per centimeter. If resolution unit is unknown, return null.
+   * @returns Object with x and y resolutions in pixel/cm.
+   */
+  get normalizedResolution() {
+    if (!this.originalResolution) {
+      return undefined;
+    }
+    const centimetersPerInch = 2.54;
+    const centimetersPerMeter = 100;
+    switch (this.originalResolution.unit) {
+      case 'inch':
+        return {
+          x: this.originalResolution.xValue / centimetersPerInch,
+          y: this.originalResolution.yValue / centimetersPerInch,
+        };
+      case 'centimeter':
+        return {
+          x: this.originalResolution.xValue,
+          y: this.originalResolution.yValue,
+        };
+      case 'meter':
+        return {
+          x: this.originalResolution.xValue / centimetersPerMeter,
+          y: this.originalResolution.yValue / centimetersPerMeter,
+        };
+      case 'unknown':
+        return null;
+      default:
+        throw new Error('Unknown resolution unit.');
+    }
+  }
+  public getOriginalResolution() {
```

Comment:
I agree with Michaël, the field can be public and this function does not need to exist.